vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/selenium/VividusMobileAppDriverFactory.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.mobileapp.selenium;
+
+import org.jbehave.core.model.Scenario;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.vividus.bdd.context.IBddRunContext;
+import org.vividus.bdd.model.MetaWrapper;
+import org.vividus.bdd.model.RunningStory;
+import org.vividus.selenium.AbstractVividusDriverFactory;
+import org.vividus.selenium.IDriverFactory;
+import org.vividus.selenium.SauceLabsCapabilityType;
+import org.vividus.selenium.VividusWebDriver;
+import org.vividus.selenium.manager.IWebDriverManagerContext;
+
+public class VividusMobileAppDriverFactory extends AbstractVividusDriverFactory
+{
+    private final IDriverFactory driverFactory;
+
+    public VividusMobileAppDriverFactory(IBddRunContext bddRunContext, IWebDriverManagerContext webDriverManagerContext,
+            IDriverFactory driverFactory)
+    {
+        super(bddRunContext, webDriverManagerContext);
+        this.driverFactory = driverFactory;
+    }
+
+    @Override
+    protected void configureVividusWebDriver(VividusWebDriver vividusWebDriver)
+    {
+        DesiredCapabilities capabilities = vividusWebDriver.getDesiredCapabilities();
+        vividusWebDriver.setWebDriver(driverFactory.getRemoteWebDriver(capabilities));
+        vividusWebDriver.setRemote(false);
+    }
+
+    @Override
+    protected void setDesiredCapabilities(DesiredCapabilities desiredCapabilities, RunningStory runningStory,
+            Scenario scenario, MetaWrapper metaWrapper)
+    {
+        desiredCapabilities.setCapability(SauceLabsCapabilityType.NAME, runningStory.getName());
```

Comment:
`NAME` - is SL specific capability, I don't think we should use it in generic modules